.github/workflows/run-tests.yml

```diff
@@ -92,6 +92,11 @@ jobs:
 
     - name: Run arch-specific UMD unit tests
       run: |
+        pwd
+        export LD_LIBRARY_PATH=$(pwd)/build/lib:$LD_LIBRARY_PATH
```

Comment:
@afuller-TT @broskoTT I don't understand why suddenly I need this hack

But looks like with this I can get the tests running.

Reply:
That suggests we're trying to use something that we haven't told CMake that we're using.  Which library is failing to be found?